core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1071,18 +1070,24 @@ write_files:
           name: cluster-admin
           apiGroup: rbac.authorization.k8s.io
 
-  - path: /srv/kubernetes/rbac/cluster-role-bindings/kube-worker.yaml
+  # Allows both `kube-worker` user and members of the `system:nodes` group
+  # to perform actions needed by the `kube-proxy` component. Once kube-proxy
+  # is migrated to DaemonSets, we could set up a ServiceAccount for it and
+  # associate this role to it instead.
+  - path: /srv/kubernetes/rbac/cluster-role-bindings/node-proxier.yaml
     content: |
         kind: ClusterRoleBinding
         apiVersion: rbac.authorization.k8s.io/v1beta1
         metadata:
-          name: kube-worker
+          name: kube-aws:node-proxier
         subjects:
           - kind: User
             name: kube-worker
```

Comment:
Can we just remove the subject for `kube-worker` here? If I understand correctly, the `system:nodes` group, which the `kube-worker` user is already associated in [the above](https://github.com/kubernetes-incubator/kube-aws/pull/618/files#diff-ef25536c536667a40b993d4d24ab7567R1012),  already seem to have all the necessary roles assigned.